doc/blog/201808-UnoUnderTheHoodIntro.md

```diff
@@ -50,18 +50,17 @@ we've built this thing. 
 
 -   **Leverage existing tools** 
 
-    -   We stand on the shoulders of  giants. Microsoft's tooling is a
-        > treat to work with: [Edit and
-        > Continue](https://docs.microsoft.com/en-us/visualstudio/debugger/edit-and-continue), [Live
-        > Visual
-        > Tree](https://docs.microsoft.com/en-us/visualstudio/debugger/inspect-xaml-properties-while-debugging),
+    -   > We stand on the shoulders of  giants. Microsoft's tooling is a
```

Comment:
list items' quotation from this list don't need  another level of bullet.

Reply:
Done